### ISSUE TITLE
Tests for WCAG technique H88: Using HTML according to spec

### ISSUE THREAD

**Issue by jessebeach** [quailjs/quail#86]
This issue will most likely result in several tests. Please open more issues and link them as appropriate.
## Testing

From http://www.w3.org/TR/WCAG20-TECHS/H88.html)
### Procedure

For each HTML or XHTML page:
- [ ] Check that the page uses only elements, attributes, and attribute values that are defined in the relevant specification.
- [ ] Check that elements, attributes, and values are used in the manner prescribed by the relevant specification.
- [ ] Check that the page can be parsed correctly, according to the rules of the relevant specification.

Note: Check #1 and #3 are most easily checked with page validation tools. Check #2 can be checked with the assistance of heuristic evaluation tools though manual judgment is usually required.
### Expected Results

Checks #1, #2, and #3 are true.

If this is a sufficient technique for a success criterion, failing this test procedure does not necessarily mean that the success criterion has not been satisfied in some other way, only that this technique has not been successfully implemented and can not be used to claim conformance.
